src/plugins/vis_type_vega/server/usage_collector/get_usage_collector.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { get } from 'lodash';
```

Comment:
I find it justified to use `lodash/get `for this case. First of all, the code is more readable and looks in accordance with `visualizations` Prefer to keep it as it

Reply:
We should not use `lodash.get` in any new code. First we nowadays have clear and good native APIs for that (and I ersonally also don't see how `get(esResponse, 'hits.hits.length', 0);` is more readable than `esResponse?.hits?.hits?.length ?? 0`). Also more important for specifically `get` - it completely disables TypeScript's type system, so you'll end up with an `any` returned from that, even in places where it wouldn't need to be, because we potentially have the correct types (and if we don't have the correct types yet, we're def planning on adding them, but those lodash.get places would still result in `any` at that time).

![screenshot-20200928-144225](https://user-images.githubusercontent.com/877229/94433638-ec892600-0198-11eb-9e4c-3dacc702e6d2.png)